src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js

```diff
@@ -44,12 +46,25 @@ define([], function() {
                 ].join('');
             },
 
-            contentItem: function(id, value) {
+            contentItem: function(id, value, icons) {
                 return [
+                    '<span class="icons">' + icons + '</span>',
                     '<a href="#" data-id="', id, '" class="link">',
                     '    <span class="value">', value, '</span>',
                     '</a>'
                 ].join('');
+            },
+
+            icons: {
+                notPublishedState: function(title) {
```

Comment:
I know these variables are called like this in the API, but why can't we at least use better names here? `onlyDraft` and `publishedAndDraft` sound already better, don't they? Or would it maybe even be better to split the two templates completely?